src/styles/cards.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-inline: 16px;
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-inline: 3px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: $main-text-color;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
```

Comment:
The selector `&:hover .card__title` might not work as expected because BEM methodology suggests using a single class for each element. Consider using a more specific class for the hover state.